routing/base/followed_polyline.hpp

```diff
@@ -115,6 +122,48 @@ class FollowedPolyline
     return res;
   }
 
+  /// \returns pair of iterator (projection point) and bool (true if nearest point is on unmatched segment)
```

Comment:
`/// \returns pair of iterator (projection point) and bool (true if the nearest point is on an unmatched segment).`?